db/migration/src/m20250219_000021_channels_add_index.rs

```diff
@@ -0,0 +1,63 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_index(
+                sea_query::Index::create()
+                    .if_not_exists()
+                    .name("idx_channel_closure_time")
+                    .table(Channel::Table)
+                    .col((Channel::ClosureTime, IndexOrder::Asc))
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
+            .create_index(
+                sea_query::Index::create()
+                    .if_not_exists()
+                    .name("idx_channel_status")
+                    .table(Channel::Table)
+                    .col((Channel::Status, IndexOrder::Asc))
+                    .to_owned(),
+            )
+            .await
+    }
+
+    async fn down(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .drop_index(Index::drop().name("idx_channel_closure_time").to_owned())
```

Comment:
Maybe the idx name should be extracted into a separate variable, to avoid typos.

Reply:
done